gatherling/report.php

```diff
@@ -10,10 +10,7 @@
     if (isset($_POST['action'])) {
         if ($_POST['action'] == 'finalize_result') {
             // write results to matches table
-            $drop = false;
-            if (isset($_POST['drop'])) {
-                $drop = $_POST['drop'] == 'Y';
-            }
+            $drop = isset($_POST['drop']);
```

Comment:
Why did you remove the value check?

Reply:
Because now you only need to check if the POST['drop'] exist or not. If it exist, means they are dropping. If not exist, means they are not dropping.

Reply:
When the checkbox isn't checked, no 'drop' value will be sent in the POST data.